front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -257,6 +259,14 @@ const SimulationReportSheet = ({
                 const isFirstStep = index === 0;
                 const isLastStep = index === operationalPointsList.length - 1;
                 const prevStep = operationalPointsList[index - 1];
+                const simulationPathStepsWithoutFirstAndLast = stdcmData.simulationPathSteps.slice(
+                  1,
+                  -1
+                );
+                const isViaInSimulationPath = simulationPathStepsWithoutFirstAndLast.some(
```

Comment:
I think you can remove this const since it is used only once
```suggestion
                const isViaInSimulationPath = stdcmData.simulationPathSteps.slice(
                  1,
                  -1
                ).some(
```

Reply:
done